### PR TITLE
Remove unused prettier format script

### PR DESCRIPTION
**Pull request by matsjla**
This script isn't used by any of our CI infrastructure, and is pretty useless because it just writes to stdout

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -168,7 +168,6 @@
     "sentry": "npx -p @sentry/cli sentry-cli",
     "update-browserslist": "npx browserslist@latest -- --update-db",
     "prepare": "husky install",
-    "format": "prettier --ignore-unknown .",
     "format-fix": "prettier --write .",
```
Comment (RubenRBS) [inline package.json:171]: Can we have `format` be the one that fixes it and drop the `-fix` naming?

**Review by RubenRBS — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (RubenRBS) [reply package.json:171]: Yes! I'd say go for it. The original names are just me making names up as I went, so I think fixing those would be great

Comment (mattgodbolt) [reply package.json:171]: Agreed here - change both please :)

File: package.json [hunk 2]
```diff
@@ -154,12 +154,12 @@
   },
   "scripts": {
     "ci-lint": "eslint --format github .",
-    "ci-lint-files": "eslint --format github",
     "ci-test": "nyc npm run test",
-    "lint": "eslint .",
-    "lint-fix": "eslint --fix .",
+    "lint": "eslint . --fix",
+    "lint-check": "eslint .",
+    "lint-files": "eslint",
```
Comment (matsjla) [inline package.json:160]: I'm assuming ci-lint-files was a typo, changed to `lint-files` to be in line with the lint/format tasks

File: package.json [hunk 3]
```diff
@@ -168,21 +168,20 @@
...
   "lint-staged": {
     "*.[jt]s": [
-      "npm run format-fix-files --",
-      "npm run ci-lint-files --"
+      "npm run format-files --",
+      "npm run lint-files --"
     ],
     "*.{html,md}": [
-      "npm run format-fix-files --"
+      "npm run format-files --"
```
Comment (matsjla) [inline package.json:184]: Changed to use common naming scheme

**Review by matsjla — COMMENTED** [review]
Tasks now do the following:

- `npm run format/lint`: Fix eslint/apply formatting fixes
- `npm run format-check/lint-check`: Test the codebase for lint/format errors
- `npm run format-files`: Format the staged files (called from lint-staged) before commit
- `npm run lint-files` Test the staged files before commit

**Review by RubenRBS — APPROVED** [review]
Lgtm

File: package.json (hunk 2 above)
Comment (RubenRBS) [reply package.json:160]: Where do you say the typo was? The ci- prefix was because only the ci called this, but happy to have more uniform naming :)

Comment (matsjla) [reply package.json:160]: Assumed it was a typo because ci-x tasks are ran on CI but there's no staged files in CI (because they're commited)

**Pull request merged**